sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -659,6 +659,16 @@ public partial class RetryOptions
         public System.TimeSpan NetworkTimeout { get { throw null; } set { } }
     }
     public delegate System.Threading.Tasks.Task SyncAsyncEventHandler<T>(T e) where T : Azure.SyncAsyncEventArgs;
+    public partial class TelemetryDetails
+    {
+        public TelemetryDetails(System.Reflection.Assembly assembly, string? applicationId = null) { }
+        public string? ApplicationId { get { throw null; } }
+        public System.Reflection.Assembly Assembly { get { throw null; } }
+        public string UserAgent { get { throw null; } }
+        public void Apply(Azure.Core.HttpMessage message) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Why are we hiding `ToString`?

Reply:
https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-editor-browsable-state but I could see this as not being considered a model type.

Reply:
ToString is no longer implemented based on an earlier suggestion of having the UserAgent property instead. But it sounds like the recommendation is to go back to ToString and remove the UserAgent property.

Reply:
In particular, this line from the guidelines:
 > Hide the ToString() method if it isn’t overridden.